test/langtools/jdk/javadoc/doclet/testJavaFX/TestJavaFX.java

```diff
@@ -374,7 +374,7 @@ public void test4() {
         checkExit(Exit.OK);
 
         // make sure the doclet indeed emits the warning
-        checkOutput(Output.OUT, true, "C.java:0: warning - invalid usage of tag <");
+        checkOutput(Output.OUT, true, "C.java:31: warning: invalid usage of tag <");
```

Comment:
This is interesting: while the line coordinate of an error has changed in this test, those of  [TestSearch.java L676-L679](https://github.com/openjdk/jdk/pull/4074/files#diff-79c641910bccd3bfabdf3912fd24f4858edcd17bdd325555930fd85fba580593L676-L679) from this same PR have not. Mind you, I haven't investigated it yet.

Reply:
It's a different error message, coming from a different place in the code, presumably with different contextual info.

In the other case, it was correct both before and after. In this case, it was wrong before and now correct. So that's good.

Tip: to find where errors come from, use (I think) `-XD-doe`.   As I wrote earlier in a different PR, `-XD` is the general mechanism to set any option. `-doe` is a hidden javac option, short for "dump on error", and triggers a track trace when any diagnostic is reported. It's handled down in Log.  Look for uses of `Log.dumpOnError`, especially roundabout line 771.